From fec268819dcc297cbe606edef0c9771682cde0fe Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 7 Oct 2023 14:25:45 -0600
Subject: [PATCH] Bump Security String to 2023-10-06

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-21244  A-276729064    EoP    High       11, 12, 12L, 13
CVE-2023-21252  A-275339978    DoS    High       11, 12, 12L, 13
CVE-2023-40116  A-270368476    EoP    High       11, 12, 12L
CVE-2023-40117  A-253043065    EoP    High       11, 12, 12L, 13
CVE-2023-40120  A-274775190    EoP    High       11, 12, 12L, 13
CVE-2023-40121  A-224771621    ID     High       11, 12, 12L, 13
CVE-2023-40123  A-278246904    ID     High       11, 12, 12L, 13
CVE-2023-40125  A-279902472    EoP    High       11, 12, 12L, 13
CVE-2023-40127  A-262244882    ID     High       11, 12, 12L, 13
CVE-2023-40128  A-274231102    EoP    High       11, 12, 12L, 13
CVE-2023-40130  A-289809991    EoP    High       11, 12, 12L, 13
CVE-2023-40133  A-283264674    ID     High       11, 12, 12L, 13
CVE-2023-40135  A-281848557    ID     High       11, 12, 12L, 13
CVE-2023-40136  A-281666022    ID     High       11, 12, 12L, 13
CVE-2023-40137  A-281665050    ID     High       11, 12, 12L, 13
CVE-2023-40138  A-281534749    ID     High       11, 12, 12L, 13
CVE-2023-40139  A-281533566    ID     High       11, 12, 12L, 13
CVE-2023-40140  A-274058082    EoP    High       11, 12, 12L, 13

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2023-4863   A-299477569    RCE    Critical   11, 12, 12L, 13         eee262a
CVE-2023-21253  A-266580022    DoS    High       11, 12, 12L, 13         c6e04b0
CVE-2023-21266  A-223376078    EoP    High       11, 12, 12L, 13         0e52101dbc3e
CVE-2023-21291  A-277593270    ID     High       11, 12, 12L, 13         6d75d088a674

Not Implemented:
=======================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-40129  A-273874525    RCE    Critical   12, 12L, 13
CVE-2023-40131  A-282919145    EoP    High       12, 12L, 13
CVE-2023-40134  A-283101289    ID     High       12, 12L, 13

Change-Id: Id4164bb314eca243bea55d7c46628d1841521ff0
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index a2fc3036da..cf9c58d1b6 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2023-09-05
+      PLATFORM_SECURITY_PATCH := 2023-10-06
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.34.1

